### PR TITLE
add test cases for custom logo

### PR DESCRIPTION
**Pull request by PrzemyslawBanasiak13**
Signed-off-by: pbanasiak <przemyslaw.banasiak@3mdeb.com>

### CODE REVIEW

File: docs/unified-test-documentation/dasharo-compatibility/328-logo-customization-functionality.md [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+1. Proceed with the
+    [Generic test setup: OS installation](../../generic-test-setup/#os-installation).
+1. Proceed with the
+    [Generic test setup: OS boot from disk](../../generic-test-setup/#os-boot-from-disk).
+
+## LCM001.001 Replace logo in existing image and flashing firmware
+
+**Test description**
+
+This test aims to verify whether it's possible to replace logo in existing
```
Comment (sulewskiprzemyslaw) [inline docs/unified-test-documentation/dasharo-compatibility/328-logo-customization-functionality.md:20]: Is the purpose of this test to flash the device with the firmware? In my opinion, the primary goal is to check if the correct logo appears on the boot screen after flashing.

File: docs/unified-test-documentation/dasharo-compatibility/328-logo-customization-functionality.md [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+**Test setup**
+
+1. Proceed with the
+    [Generic test setup: firmware](../../generic-test-setup/#firmware).
+1. Proceed with the
+    [Generic test setup: OS installer](../../generic-test-setup/#os-installer).
+1. Proceed with the
+    [Generic test setup: OS installation](../../generic-test-setup/#os-installation).
+1. Proceed with the
+    [Generic test setup: OS boot from disk](../../generic-test-setup/#os-boot-from-disk).
```
Comment (sulewskiprzemyslaw) [inline docs/unified-test-documentation/dasharo-compatibility/328-logo-customization-functionality.md:14]: In this section, we should add information, that familiarizing with the logo replacement procedure is necessary.

File: docs/unified-test-documentation/dasharo-compatibility/328-logo-customization-functionality.md [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+
+1. Proceed with the
+    [Test cases common documentation](#test-cases-common-documentation) section.
+
+**Test steps**
+
+1. Power on the DUT.
+1. Boot into the system.
+1. Log into the system by using the proper login and password.
+1. Proceed with the
```
Comment (sulewskiprzemyslaw) [inline docs/unified-test-documentation/dasharo-compatibility/328-logo-customization-functionality.md:69]: At this point, we should explain briefly, what operations should be performed. Linking to the documentation without mentioning, what part of it should be used might confuse the tester.

File: docs/unified-test-documentation/dasharo-compatibility/328-logo-customization-functionality.md [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+    [Build image with custom logo](../../common-coreboot-docs/custom_logo.md#build-image-with-custom-logo)
+    section.
+1. Now you can flash the updated firmware image as usual. If you're not updating
```
Comment (sulewskiprzemyslaw) [inline docs/unified-test-documentation/dasharo-compatibility/328-logo-customization-functionality.md:72]: Documentation should be written in an impersonal form, i. e. `flash the firmware (only BOOTSPLASH region`, not `you can do smth...`.

File: docs/unified-test-documentation/dasharo-compatibility/328-logo-customization-functionality.md [hunk 5]
```diff
@@ -0,0 +1,125 @@
...
+
+1. Proceed with the
+    [Test cases common documentation](#test-cases-common-documentation) section.
+
+**Test steps**
+
+1. Power on the DUT.
+1. Boot into the system.
+1. Log into the system by using the proper login and password.
+1. Proceed with the
```
Comment (sulewskiprzemyslaw) [inline docs/unified-test-documentation/dasharo-compatibility/328-logo-customization-functionality.md:38]: At this point, we should explain briefly, what operations should be performed. Linking to the documentation without mentioning, what part of it should be used might confuse the tester.

File: docs/unified-test-documentation/dasharo-compatibility/328-logo-customization-functionality.md [hunk 6]
```diff
@@ -0,0 +1,125 @@
...
+    [Build image with custom logo](../../common-coreboot-docs/custom_logo.md#build-image-with-custom-logo)
+    section, but use image that doesn't meet
+    [Prequsities](../../common-coreboot-docs/custom_logo.md#prerequisites).
+1. Now you can flash the updated firmware image as usual. If you're not updating
```
Comment (sulewskiprzemyslaw) [inline docs/unified-test-documentation/dasharo-compatibility/328-logo-customization-functionality.md:112]: Documentation should be written in an impersonal form, i. e. `flash the firmware (only BOOTSPLASH region`, not `you can do smth...`.

File: docs/unified-test-documentation/dasharo-compatibility/328-logo-customization-functionality.md [hunk 7]
```diff
@@ -0,0 +1,125 @@
...
+
+**Expected result**
+
+1. While booting custom logo appears on screen.
+
+## LCM002.001 Build image with custom logo and flashing firmware
+
+**Test description**
+
+This test aims to verify whether it's possible to build firmware image from the
```
Comment (sulewskiprzemyslaw) [inline docs/unified-test-documentation/dasharo-compatibility/328-logo-customization-functionality.md:51]: Is the purpose of this test to flash the device with the firmware? In my opinion, the primary goal is to check if the correct logo appears on the boot screen after flashing.

**Review by sulewskiprzemyslaw — CHANGES_REQUESTED** [review]

File: docs/unified-test-documentation/dasharo-compatibility/328-logo-customization-functionality.md (hunk 4 above)
Comment (PrzemyslawBanasiak13) [reply docs/unified-test-documentation/dasharo-compatibility/328-logo-customization-functionality.md:72]: Copied it from building custom logo from source for consistency

File: docs/unified-test-documentation/dasharo-compatibility/328-logo-customization-functionality.md (hunk 5 above)
Comment (PrzemyslawBanasiak13) [reply docs/unified-test-documentation/dasharo-compatibility/328-logo-customization-functionality.md:38]: Whole section should be performed

File: docs/unified-test-documentation/dasharo-compatibility/328-logo-customization-functionality.md (hunk 6 above)
Comment (PrzemyslawBanasiak13) [reply docs/unified-test-documentation/dasharo-compatibility/328-logo-customization-functionality.md:112]: Copied from another docs

File: docs/unified-test-documentation/dasharo-compatibility/328-logo-customization-functionality.md (hunk 2 above)
Comment (PrzemyslawBanasiak13) [reply docs/unified-test-documentation/dasharo-compatibility/328-logo-customization-functionality.md:14]: Logo replacement procedure is mentioned later in test steps as it is main part of the test

File: docs/unified-test-documentation/dasharo-compatibility/328-logo-customization-functionality.md [hunk 8]
```diff
@@ -0,0 +1,125 @@
...
+    [Generic test setup: OS installation](../../generic-test-setup/#os-installation).
+1. Proceed with the
+    [Generic test setup: OS boot from disk](../../generic-test-setup/#os-boot-from-disk).
+
+## LCM001.001 Replace logo in existing image and flashing firmware
+
+**Test description**
+
+This test aims to verify whether it's possible to replace logo in existing
+firmware image and flash this firmware on the DUT.
```
Comment (PrzemyslawBanasiak13) [inline docs/unified-test-documentation/dasharo-compatibility/328-logo-customization-functionality.md:21]:
```suggestion
firmware and after flash it shows properly.
```

Comment (PrzemyslawBanasiak13) [inline docs/unified-test-documentation/dasharo-compatibility/328-logo-customization-functionality.md]:
```suggestion
firmware and after flash it shows properly.
```

File: docs/unified-test-documentation/dasharo-compatibility/328-logo-customization-functionality.md (hunk 3 above)
Comment (PrzemyslawBanasiak13) [reply docs/unified-test-documentation/dasharo-compatibility/328-logo-customization-functionality.md:69]: Whole section should be performed

File: docs/unified-test-documentation/dasharo-compatibility/328-logo-customization-functionality.md (hunk 1 above)
Comment (PrzemyslawBanasiak13) [reply docs/unified-test-documentation/dasharo-compatibility/328-logo-customization-functionality.md:20]: changed description

File: docs/unified-test-documentation/dasharo-compatibility/328-logo-customization-functionality.md (hunk 4 above)
Comment (PrzemyslawBanasiak13) [reply docs/unified-test-documentation/dasharo-compatibility/328-logo-customization-functionality.md:72]: Changed

File: docs/unified-test-documentation/dasharo-compatibility/328-logo-customization-functionality.md (hunk 6 above)
Comment (PrzemyslawBanasiak13) [reply docs/unified-test-documentation/dasharo-compatibility/328-logo-customization-functionality.md:112]: Changed

File: docs/unified-test-documentation/dasharo-compatibility/328-logo-customization-functionality.md (hunk 7 above)
Comment (PrzemyslawBanasiak13) [reply docs/unified-test-documentation/dasharo-compatibility/328-logo-customization-functionality.md:51]: Changed

**Review by sulewskiprzemyslaw — APPROVED** [review]

**Pull request merged**